### DISCUSSION TITLE
I get this for web view - how do I show the calendar?

### DISCUSSION THREAD

**Discussion by sigkappel** [YukiGasai/obsidian-google-calendar#125] [Q&A] [answered]
When I use this embed command

```gEvent
type: web
```

I'm getting the google workspace intro page instead of the calendar. 
How do I get the calendar view to show the web calendar. 

![image](https://github.com/YukiGasai/obsidian-google-calendar/assets/5200412/ea7ca189-f632-4e1d-876c-b136e695b305)

**Comment by YukiGasai** [comment]
It looks like you are not signed into a google account. Please try to log in and restart obsidian.
If the Sign-in button doesn't work, you can go into the view settings.
![image](https://github.com/YukiGasai/obsidian-google-calendar/assets/38146192/540cb4fc-6d0b-4782-9919-5beac68c0c56)
and press open Login.

**Comment by sigkappel** [comment]
Ok tried it. not working. 
here's a screen recording

https://vimeo.com/830240716/6396f26bf1

**Comment by YukiGasai** [comment, marked as answer]
Hi, 
sorry for the miss communication.
The web view is completely separate from the usual plugin login.
It is just a window to the Google calendar website. It's like an extra browser window and you have to log in to your Google account inside this window. 
The problem is that this window can't open a link in a new tab, that's why the login button won't work. To work around this, you can press the login button in the web view  settings to navigate the window to the login URL. 

https://github.com/YukiGasai/obsidian-google-calendar/assets/38146192/21e34d93-780d-4723-af17-e124287ecde7

**Comment by sigkappel** [comment]
Yesss! Bro! thank you works now
![image](https://github.com/YukiGasai/obsidian-google-calendar/assets/5200412/f50a9498-1f23-460a-84c1-ef6147c4958a)
